sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -167,4 +185,37 @@ public class ChatClient {
             }
         }
     }
+
+    /// Start receiving realtime notifications.
+    /// Call this function before subscribing to any event.
+    public func startRealTimeNotifications () {
+        if self.signalingClient == nil {
+            print("no signaling client is initialized")
+        }
```

Comment:
Should use logger. Is this a warning? Error? info? Should this be a guard statement?